assets/scss/_custom.scss

```diff
@@ -578,3 +578,64 @@ body.td-documentation {
   color: black;
   text-decoration: none !important;
 }
+
+@media print {
+  /* Do not print announcements */
+  #announcement, section#announcement, #fp-announcement, section#fp-announcement {
+    display: none;
+  }
+}
+
+#announcement, #fp-announcement {
+  > * {
+    color: inherit;
+    background: inherit;
+  }
+
+  a {
+    color: inherit;
+    border-bottom: 1px solid #fff;
+  }
+
+  a:hover {
+    color: inherit;
+    border-bottom: none;
+  }
+}
+
+#announcement {
+  padding-top: 105px;
+  padding-bottom: 25px;
+}
+
+.header-hero {
+  padding-top: 40px;
+}
```

Comment:
this doesn't seem scoped to when the announcement is showing

Reply:
(it also seemed to be overridden by other padding rules... removing it didn't seem to change anything for frontpage or other page views, with or without an announcement displaying)